lib/ansible/modules/network/aci/aci_l3out.py

```diff
@@ -234,16 +239,17 @@ def main():
                   choices=['AF11', 'AF12', 'AF13', 'AF21', 'AF22', 'AF23', 'AF31', 'AF32', 'AF33', 'AF41', 'AF42',
                            'AF43', 'CS0', 'CS1', 'CS2', 'CS3', 'CS4', 'CS5', 'CS6', 'CS7', 'EF', 'VA', 'unspecified'],
                   aliases=['target']),
-        l3protocol=dict(type='list', choices=['static', 'bgp', 'ospf', 'pim']),
+        l3protocol=dict(type='list', choices=['static', 'bgp', 'ospf', 'eigrp', 'pim']),
```

Comment:
Can you sort them alphabetically ?

Reply:
No problem at all, it wasn't sorted to begin with.
If the order is of no relevance, it's always better to sort alphabetically as it improves readability.